config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```diff
@@ -99,7 +99,7 @@ presubmits:
     optional: true
     run_if_changed: 'capz/gmsa/.*|capz/templates/gmsa.yaml'
     decoration_config:
-      timeout: 3h
```

Comment:
Not a _serial-slow_ job. Please lemme know if we want to revert `pull-e2e-capz-containerd-windows-2022-extension-gmsa` timeout to `3h`